processing/src/main/java/io/druid/segment/QueryableIndexIndexableAdapter.java

```diff
@@ -365,41 +363,13 @@ public BitmapIndexSeeker getBitmapIndexSeeker(String dimension)
     // By using this and the help of "getDimValueLookup", we only need to translate all dimension value to its ID once,
     // and the translation is done by self-increase of the integer. We only need to change the CACHED value once after
     // previous value is hit, renew the value and increase the ID. The complexity now is O(N * M * LOG(M)).
-    return new BitmapIndexSeeker()
+    return new BitmapIndexHolder()
```

Comment:
given that BitmapIndexHolder doesn't do very much anymore, do we still need the interface?